quadriga/src/main/java/edu/asu/diging/quadriga/web/AddAppController.java

```diff
@@ -0,0 +1,51 @@
+package edu.asu.diging.quadriga.web;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.quadriga.core.exceptions.UnstorableObjectException;
+import edu.asu.diging.quadriga.core.service.IAppManager;
+import edu.asu.diging.quadriga.web.forms.AppForm;
+
+
+@Controller
+public class AddAppController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+    @Autowired
+    private IAppManager appManager;
+
+
+    @RequestMapping(value = "/admin/add", method=RequestMethod.GET)
```

Comment:
since this is not adding an admin, i would suggest `/admin/apps/add` or something like that ;)